ci-operator/step-registry/openshift/installer/manifests/aws/edge-zones/openshift-installer-manifests-aws-edge-zones-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: openshift-installer-manifests-aws-edge-zones
+  steps:
+    env:
+      BASE_DOMAIN: vmc-ci.devcluster.openshift.com
+    allow_best_effort_post_steps: true
+    pre:
+    - chain: ipi-conf-aws
+    test:
+    - ref: openshift-installer-manifests-aws-edge-zones
+  documentation: |-
+    The Openshift E2E AWS workflow executes the common end-to-end test suite on AWS with a default cluster configuration.
```

Comment:
This doc seems a c&p from another step. It would be good to update it since we're not actually running e2e tests in this workflow.